Dockerfile.local

```diff
@@ -17,18 +15,6 @@ RUN git clone https://github.com/commonmark/cmark \
 RUN make -C cmark INSTALL_PREFIX=/usr
 RUN make -C cmark install
 
-RUN swift build --configuration release --product Server -Xswiftc -g -Xswiftc -Xfrontend -Xswiftc -warn-long-function-bodies=200 -Xswiftc -Xfrontend -Xswiftc -warn-long-expression-type-checking=200 \
-  && swift build --configuration release --product Runner -Xswiftc -g -Xswiftc -Xfrontend -Xswiftc -warn-long-function-bodies=200 -Xswiftc -Xfrontend -Xswiftc -warn-long-expression-type-checking=200
+RUN swift build --build-tests --jobs 1 -Xswiftc -D -Xswiftc OSS
```

Comment:
We should investigate this or submit a bug, but switching to `--jobs 1` is what prevented the weird Docker exits I was seeing.